lib/challenge.js

```diff
@@ -51,7 +66,15 @@ var pet_owner;
 var some_pet;
 
 // Answer Starts Here
-
+some_pet = {
+  name: "Brownie",
+  species: "dog",
+  breed: "Border collie",
+  noise: "bark"
```

Comment:
A comma is needed here to separate this from the next key-value pair. Did your linter highlight this?